clients/admin-ui/src/features/common/nav/v2/nav-config.test.ts

```diff
@@ -56,8 +56,9 @@ describe("configureNavGroups", () => {
       config: NAV_CONFIG,
       hasSystems: true,
       userScopes: [
-        ScopeRegistry.CLI_OBJECTS_CREATE,
-        ScopeRegistry.CLI_OBJECTS_READ,
+        ScopeRegistryEnum.SYSTEM_CREATE,
```

Comment:
I think this should be reverted back to CLI_OBJECTS_CREATE - maybe DATASET_CREATE_OR_UPDATE can stay